script.js

```diff
@@ -1,4 +1,398 @@
+    if (
+      myFirstOutputValue == "You shouted: muk-jji-ppa!" ||
+      myFirstOutputValue == "Computer shouted: muk-jji-ppa!"
+    ) {
+      return myFirstOutputValue;
+    }
+    var mySecondOutputValue =
+      winningPercentageOutputForComputerVSComputer(totalCount);
+    return `${standardOutput} ${myFirstOutputValue}<br>${mySecondOutputValue}`;
+  }
+};
+
+var main = function (input) {
+  if (currentGameMode == "user to input username") {
+    userName = input;
+    currentGameMode = "choose player mode";
+    return `Welcome ${userName}!<br><br>This is a game of "Scissors, Paper, Stone"!<br><br>Please enter "Mode 1: Player VS Computer" or "Mode 2: Computer VS Computer". Thank you!`;
+  }
+  if (currentGameMode == "choose player mode") {
+    playerOrComputerMode = input; // input: "Mode 1: Player VS Computer" or "Mode 2: Computer VS Computer"
+    currentGameMode = "choose normal or korean version";
+    return `Hi ${userName}, you have chosen ${playerOrComputerMode}.<br><br>Please enter "Normal Version" or "Korean Version".`;
+  }
+  if (
+    currentGameMode == "choose normal or korean version" &&
+    playerOrComputerMode == "Mode 1: Player VS Computer"
+  ) {
+    normalOrKoreanVersion = input; // input: "Normal Version" or "Korean Version"
+    currentGameMode = "game start";
+    return `Hi ${userName}, you have chosen the ${normalOrKoreanVersion}.<br><br>Please enter "scissors" "paper" or "stone"!<br>You may add a "reversed" infront of your choice too! e.g. "reversed scissors"`;
+  }
+  if (
+    currentGameMode == "choose normal or korean version" &&
+    playerOrComputerMode == "Mode 2: Computer VS Computer"
+  ) {
+    normalOrKoreanVersion = input; // input: "Normal Version" or "Korean Version"
+    currentGameMode = "game start";
+    return `Hi ${userName}, you have chosen the ${normalOrKoreanVersion}.<br><br>Please click "Submit" so as to start the Computer VS Computer mode!`;
+  }
+  if (
+    playerOrComputerMode == "Mode 1: Player VS Computer" &&
+    normalOrKoreanVersion == "Normal Version"
+  ) {
+    var mode = "Mode 1: Player VS Computer";
+    var output = normalVersion(mode, input);
+    return output;
+  }
+  if (
+    playerOrComputerMode == "Mode 2: Computer VS Computer" &&
+    normalOrKoreanVersion == "Normal Version"
+  ) {
+    var mode = "Mode 2: Computer VS Computer";
+    input = "null";
+    var output = normalVersion(mode, input);
+    return output;
+  }
+  if (
+    playerOrComputerMode == "Mode 1: Player VS Computer" &&
+    normalOrKoreanVersion == "Korean Version"
+  ) {
+    var mode = "Mode 1: Player VS Computer";
+    var output = koreanVersion(mode, input);
+    return output;
+  }
+  if (
+    playerOrComputerMode == "Mode 2: Computer VS Computer" &&
+    normalOrKoreanVersion == "Korean Version"
+  ) {
+    var mode = "Mode 2: Computer VS Computer";
+    input = "null";
+    var output = koreanVersion(mode, input);
+    return output;
+  }
+};
```

Comment:
I like how you have separated out your logic into helper functions instead of placing them all in your main function. Good work!